recipe-box-frontend/src/main/frontend/src/components/RecipeView/RecipeViewGrid.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import RecipeDetails from './RecipeDetails';
+import RecipeImageDetails from './RecipeImageDetails';
+import { Recipe } from './types';
+import "./RecipeViewGrid.css";
+
+const RecipeViewGrid: React.FC = () => {
+    const [recipe, setRecipe] = useState<Recipe | null>(null);
+    const [error, setError] = useState<string | null>(null);
+    const [loading, setLoading] = useState<boolean>(true);
+
+    useEffect(() => {
+        const recipeId = 10;
+        const userId = 1;
+
+        fetch(`http://localhost:8080/recipes/${recipeId}?userId=${userId}`)
```

Comment:
Here we could use the env var "REACT_APP_BACKEND_URL" (called backendUrl in App.tsx)